### PR TITLE
Feature/fixes for variables pr

### PR DESCRIPTION
**Pull request by vgro**
This PR adds missing variables to the data object and updates some units in preparation for #388 

## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Optimization (back-end change that speeds up the code)
- [x] Bug fix (non-breaking change which fixes an issue)

## Key checklist

- [x] Make sure you've run the `pre-commit` checks: `$ pre-commit run -a`
- [x] All tests pass: `$ poetry run pytest`

## Further checks

- [x] Code is commented, particularly in hard-to-understand areas
- [x] Tests added that prove fix is effective or that feature works

### CODE REVIEW

File: virtual_ecosystem/models/abiotic/wind.py [hunk 1]
```diff
@@ -630,8 +642,7 @@ def calculate_wind_profile(
         core_constants: Universal constants shared across all models
 
     Returns:
-        Dictionnary that contains wind speed and friction velocity, [m s-1] and
-        diabatic correction factors for heat and momentum transfer
+        Dictionnary that contains wind related outputs
```
Comment (jacobcook1995) [inline virtual_ecosystem/models/abiotic/wind.py:645]: "Dictionary"

**Review by jacobcook1995 — APPROVED** [review]
LGTM! (just noticed one typo)

**Comment by davidorme** [comment]
Should this merge into the `variables` branch, not `develop`?

File: virtual_ecosystem/models/abiotic/wind.py (hunk 1 above)
Comment (vgro) [inline virtual_ecosystem/models/abiotic/wind.py:645]:
```suggestion
        Dictionary that contains wind related outputs
```

**Pull request merged**